docs/StandaloneConfiguration.md

```diff
@@ -1,8 +1,33 @@
 # RepoSense - Guide to Setup Standalone Configuration
-
+### Quick Start
 1. Add [_reposense/config.json](../_reposense/config.json) to the root of your repository.
-1. Fill it up with author information, such as the GitHub Id, Display Name (Optional) and the [Author Names](UserGuide.md#git-author-name) (Optional).
+1. Fill it up with author information, such as the GitHub Id, Display Name (Optional), [Author Names](UserGuide.md#git-author-name)  (Optional), [Ignore Glob List](UserGuide.md#csv-config-file) (Optional).
+
+### Detailed explanation of the structure of repo-config.json
+```
+{
+  "ignoreGlobList": ["**.dat", "**.js"],    <-- Repository level's list of file formats to ignore 
+  "authors": [                              <-- If a setting is common between Repository and Author,
+  {                                             configuration of Author level will always take precedence
+    "githubId": "alice",                        over Repository level.
+    "displayName": "Alice T.",
+    "authorNames": ["AT", "A"],
+    "ignoreGlobList": [""]                  <-- Author level's list of file formats to ignore.
+  },                                            Use "" to override Repository level settings.
```

Comment:
`""` alone doesn't override the repo level settings, but rather any non-empty `ignoreGlobList` right?
Maybe can say use `""` if we wish to ignore the repo level setting instead?